workers/generic-worker/gdm3/gdm3.go

```diff
@@ -57,6 +52,9 @@ func SetAutoLogin(username string, source []byte) (output []byte) {
 					"# Set by generic-worker",
 					"AutomaticLoginEnable = true",
 					"AutomaticLogin = "+username,
+					"TimedLoginEnable = true",
+					"TimedLogin = "+username,
+					"TimedLoginDelay = 10",
```

Comment:
does this number has any background, maybe worth living some context why `10`
do you know if it is something performance related? If we allow to run it on a slower VM, would it take longer than this for gdm to do its job and prevent race condition?